recipes/newrelic/apm/php/debian-fpm.yml

```diff
@@ -204,16 +204,28 @@ install:
           cd {{.TMP_INSTALL_DIR}}
           # Remove old log tarballs to ensure we don't accidentally use it later
           rm -f /tmp/nrinstall* 2>/dev/null
-          AGENT="newrelic-php5-{{.AGENT_VERSION}}-linux"
+
+          # Versions of the form MAJ.MIN.PATCH.BUILD
+          VERSION_REGEX='[1-9][0-9]\?\(\.[0-9]\+\)\{3\}'
+          # PHP Agent current release tarball listing
+          RELEASE_URL='https://download.newrelic.com/php_agent/release/'
+          AGENT_VERSION="$(curl -s "$RELEASE_URL" | grep --only-match "$VERSION_REGEX" | head -n1)"
+          if [ -z "${AGENT_VERSION}" ]; then
+            echo -e "{{.RED}}Unable to determine current PHP Agent version from New Relic's downloads site.{{.NOCOLOR}}"
+            # Exit with Agent install failure code exit(16)
+            exit 16
+          else
+            echo "Found agent version: $AGENT_VERSION"
```

Comment:
Consider outputting this message in white and the agent version in cyan. Which version of the agent we are installing feels like something worth highlighting.

If you do implement this, you'll need to add this var:
```
CYAN: '\033[0;36`
```